app/forms/idv/api_image_upload_form.rb

```diff
@@ -370,7 +370,7 @@ def add_costs(response)
       Db::AddDocumentVerificationAndSelfieCosts.
         new(user_id: user_id,
             service_provider: service_provider,
-            liveness_checking_enabled: liveness_checking_enabled).
+            liveness_checking_enabled: liveness_checking_required?).
```

Comment:
Should we check the costs function actually adds the cost?

Reply:
you're right .. we shouldn't forget about this ... i think that's out of scope for this ticket ... created LG-11965 [dt](https://cm-jira.usa.gov/browse/LG-11965) so we won't lose track of this ... 👍🏿